man/cve.1

```diff
@@ -30,12 +30,24 @@ Confirm create/update actions before execution (env var: CVE_INTERACTIVE)
 Show the version and exit.
 .SH COMMANDS
 .PP
+\fBpublish\fP
+  Publish a CVE record for a reserved (or...
+  See \fBcve-publish(1)\fP for full documentation on the \fBpublish\fP command.
+.PP
+\fBreject\fP
+  Reject a CVE record for a reserved or...
+  See \fBcve-reject(1)\fP for full documentation on the \fBreject\fP command.
+.PP
+\fBundo-reject\fP
+  Move a rejected CVE ID without a record...
+  See \fBcve-undo-reject(1)\fP for full documentation on the \fBundo-reject\fP command.
+.PP
```

Comment:
Did we miss "move-to-rejected" in docs?

Reply:
Which "docs" do you mean? The README file?

Reply:
This man/cve.1 file here - I see a diff with lines added for all other commands, but not for undo reject.

Reply:
That's only an interface method, the `reject` command can take no record and works as the reserved-to-rejected without a record state switch. `undo-reject` is the only special command that has to support the transition back to reserved.